wallet-api/src/Wallet/Emulator/Types.hs

```diff
@@ -317,7 +315,7 @@ emptyEmulatorState = EmulatorState {
     _chainNewestFirst = [],
     _txPool = [],
     _walletStates = Map.empty,
-    _index = Index.empty,
+    _index = mempty,
```

Comment:
`EmulatorState` can probably derive `Monoid` now and you can ditch this.

Reply:
it doesn't have a good semigroup instance though because you can't just append two 'Blockchain's.

Reply:
Fair.